src/service/PoolService.ts

```diff
@@ -15,9 +15,26 @@ class PoolService {
     return await poolRepository.save(pool);
   }
 
-  // async getPoolById(id: number): Promise<Pool> {
-  //   return await poolRepository.findOne(id);
-  // }
+  async getPoolById(id: number): Promise<Pool> {
```

Comment:
we need to think about a smart way to communicate, that these are the database id's and not the on chain pool id's